spec/bolt_server/app_integration_spec.rb

```diff
@@ -60,7 +60,12 @@ def app
       expect(last_response.status).to eq(200)
```

Comment:
I think we can also delete the `"message": "Hello!"` line since shareable doesn't have parameters